package.json

```diff
@@ -41,6 +41,7 @@
     "@angular/platform-browser-dynamic": "6.0.3",
     "@angular/router": "6.0.3",
     "core-js": "2.5.7",
+    "ngx-row-accordion": "^2.1.0",
```

Comment:
Can you revert the changes on package.json?

Reply:
Reverted file to previous version.